snow/engine/snowman/transitive_test.go

```diff
@@ -170,7 +170,7 @@ func TestEngineDropsAttemptToIssueBlockAfterFailedRequest(t *testing.T) {
 	// job blocked on [parent]'s issuance.
 	require.NoError(engine.Put(context.Background(), peerID, 0, child.Bytes()))
 	require.NotNil(request)
-	require.Len(engine.blocked, 1)
+	require.Equal(1, engine.blocked.NumDependencies())
```

Comment:
Since you're already updating it, consider `assert.Equal` (see rationale in earlier comment).